ios/Raha/Raha.entitlements

```diff
@@ -4,5 +4,12 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:getraha.app.link</string>
+		<string>applinks:getraha-alternate.app.link</string>
```

Comment:
Do we want all these test/alternate links in here? Do they pose a security risk?

Reply:
alternate is a hard requirement by branch, and they do the same thing in the branch docs for the test ones too.

Reply:
No security risk; if someone follows a test link it will just get attributed to Branch's Test dashboard. We still own the Test configs/private key fully. Branch's Test is a way for us to test out install attributions without affecting our Live dashboard.